runtime/kusama/src/xcm_config.rs

```diff
@@ -154,6 +154,10 @@ impl xcm_executor::Config for XcmConfig {
 
 parameter_types! {
 	pub const CouncilBodyId: BodyId = BodyId::Executive;
+	// StakingAdmin pluralistic body.
+	pub const StakingAdminBodyId: BodyId = BodyId::Defense;
+	// Fellows pluralistic body.
+	pub const FellowsBodyId: BodyId = BodyId::Technical;
```

Comment:
Would we want a new body ID for the Fellowship, or use [Technical](https://github.com/paritytech/polkadot/pull/6273/files#diff-d28701bf6e11c00b43a9393facee9862b2d4f790bf2952a511913952c845de61R47) for the Technical Committee and Fellowship (for the time that they co-exist)?
```
/// The unambiguous technical body (for Polkadot, this would be the Technical Committee).
Technical,
```

Reply:
I think it's better to use Technical. XCM is a higher level protocol, we make less changes to it, and it does not have to be one to one mapping. 
I did not touched the docs of Technical, since its only for Kusama now.

Reply:
Double checked, Technical not used/mapped right now in kusama or polkadot (only docs tells as its supposed to be used as Tech Comm), it's safe to start using it as Fellows.